app/src/main/java/com/example/vybe/AddEdit/AddEditVibeEventActivity.java

```diff
@@ -17,7 +17,7 @@
 
 import com.bumptech.glide.Glide;
 import com.example.vybe.MapFragment;
-import com.example.vybe.Models.SocialSituation;
+import com.example.vybe.Models.SocSit;
```

Comment:
wait why is this renamed to `SocSit` I thought we were just wanting to change all the variable stuff? Unless this has to do with how everything is stored in the database?

Reply:
I just changed everything for consistency

Reply:
I mean I guess that makes sense